src/styles/sidebar/components/top-bar.scss

```diff
@@ -24,7 +26,7 @@
 .top-bar__login-links {
   flex-shrink: 0;
   font-size: $body2-font-size;
-  padding-left: 6px;
+  padding-left: 5px;
```

Comment:
Bringing this into the multiples-of-5 rhythm that is at least informally the standard of the client (it looks just fine/maybe better).